src/applications/simple_storage_starknetjs.md

```diff
@@ -0,0 +1,259 @@
+# Simple Storage (Starknet-js + Cairo)
+
+In this tutorial, you will write and deploy a SimpleStorage Cairo contract on Starknet Sepolia Testnet. Then, you will learn how you can interact with the contract using Starknet-js. 
+
+In order to fully understand and implement the content of this tutorial, here are the prerequisites:
+- Basic knowledge of Cairo: you can get familiar with the Cairo language, and learn how to build a simple smart contract with Cairo [here](https://book.cairo-lang.org/).
+
+- Scarb for compiling Cairo code and packaging support: follow [here](https://docs.swmansion.com/scarb/download.html).
+
+- Starkli for the declaration and deployment of Cairo contracts: follow [here](https://book.starkli.rs/installation).
+
+## Writing SimpleStorage contract in Cairo
+The SimpleStorage contract has only one purpose: storing a number. We want the users to interact with the stored number by **viewing** the current stored number and **setting** a new number.
+
+Let's create a Scarb package in order to write and compile the SimpleStorage contract. Make sure that Scarb is installed (installation link provided in the prerequisites):
+
+```console
+//move to the directory of your choice
+$ scarb new simple_storage
+```
+
+With this command, a template for Cairo development environment is generated. Under the `src` directory, `lib.cairo` file is created. In order to start writing our contract, follow these steps: 
+- Under the `src` directory, create a file named `storage.cairo` where we will write our contract.
+- Delete the content of the `lib.cairo` file, and add `storage.cairo` as a module for the compilation of our Cairo contract: 
+```rs
+// Directory: src/lib.cairo 
+mod storage;
+```
+- You can copy and paste the following Cairo code in `storage.cairo`. In this [link](https://book.cairo-lang.org/ch13-02-anatomy-of-a-simple-contract.html), you can find explanations for each component of the contract:
+
```

Comment:
Again, format must be straightforward (example, not full tutorial).
So you can remove this part, and just state "We will use the following contract..."